plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -75,12 +75,13 @@
                                 {% if action.type == 'action' %}
 {# white spacing matters as Chrome tooltip display whitespaces #}
 {% set title %}
-{% if action.pageTitle is not empty %}{{ action.pageTitle }}{% endif %}
+{% if action.url|trim is not empty %}<span class='tooltip-action-url'>{{ action.url }}</span>{% endif %}
+    {% if action.pageTitle is not empty %}<span class='tooltip-action-page-title'>{{ action.pageTitle }}</span>{% endif %}
 
-{{ action.serverTimePretty }}
-{% if action.timeSpentPretty is defined %}{{ 'General_TimeOnPage'|translate }}: {{ action.timeSpentPretty|raw }}{% endif %}
+<span class="tooltip-action-server-time">{{ action.serverTimePretty }}</span>
+    {% if action.timeSpentPretty is defined %}<span class='tooltip-time-on-page'>{{ 'General_TimeOnPage'|translate }}: {{ action.timeSpentPretty|raw }}</span>{% endif %}
 {%- endset %}
-                                    <img class='iconPadding' src="{{ action.icon }}" title="{% if action.url|trim is not empty %}{{ action.url|e('html_attr') }}<br/>{% endif %}{{- title -}}"/>
```

Comment:
Actually the `<br/>` between url and title got lost with adding the `<span>`s. But guess it would be better to solve the line break with CSS now.